exporters/otlp/test/otlp_http_log_exporter_test.cc

```diff
@@ -103,9 +103,13 @@ class OtlpHttpLogExporterTestPeer : public ::testing::Test
     std::string attribute_storage_string_value[] = {"vector", "string"};
 
     auto provider = nostd::shared_ptr<sdk::logs::LoggerProvider>(new sdk::logs::LoggerProvider());
-    provider->AddProcessor(
-        std::unique_ptr<sdk::logs::LogProcessor>(new sdk::logs::BatchLogProcessor(
-            std::move(exporter), 5, std::chrono::milliseconds(256), 5, is_async)));
+    provider->AddProcessor(std::unique_ptr<sdk::logs::LogProcessor>(
+        new sdk::logs::BatchLogProcessor(std::move(exporter), 5, std::chrono::milliseconds(256), 5
+#    ifdef ENABLE_ASYNC_EXPORT
```

Comment:
Could we add a CI build for `ENABLE_ASYNC_EXPORT`?

Reply:
Yes, I will add on this PR